<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>761</width>
      <height>481</height>
     </rect>
    </property>
    <widget class="QWidget" name="stackedWidgetPage1" native="true">
     <widget class="QLabel" name="inputImg">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>251</width>
        <height>261</height>
       </rect>
      </property>
      <property name="text">
       <string>Input</string>
      </property>
     </widget>
     <widget class="QLabel" name="outputImg">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>251</width>
        <height>261</height>
       </rect>
      </property>
      <property name="text">
       <string>Output</string>
      </property>
     </widget>
     <widget class="QPushButton" name="input_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>40</y>
        <width>160</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="optimal">
         <property name="text">
          <string>Optimal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="otsu">
         <property name="text">
          <string>Otsu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mean_shift">
         <property name="text">
          <string>Mean Shift</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="k_means">
         <property name="text">
          <string>K-Means</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="local">
         <property name="text">
          <string>Local Thresholding</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="agglo">
         <property name="text">
          <string>Agglomerative</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="spectral">
         <property name="text">
          <string>Spectral Thresholding</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="region">
         <property name="text">
          <string>Region Growing</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>384</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_1">
         <property name="text">
          <string>Mode 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>Mode 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>Mode 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>Mode 4</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="stackedWidgetPage2" native="true"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
